fullscreen.bs

```diff
@@ -348,15 +359,10 @@ attribute's getter must run these steps:
    <var>doc</var>.
    <!-- cross-process -->
 
-   <li><p>If <var>resize</var> is true and <var>topLevelDoc</var> is not in <var>exitDocs</var>,
+   <li><p>If <var>resize</var> is true and <var>topLevelDoc</var> is either not in
+   <var>exitDocs</var>, or not a <a>simple fullscreen document</a>,
    <a>fully exit fullscreen</a> <var>topLevelDoc</var>, reject <var>promise</var> with a
```

Comment:
This is what Blink does as well. I think this change is fine, I'll try to bring it closer to implementation in https://github.com/whatwg/fullscreen/pull/65 and ask for your review then.